db/migrate/20160403055927_create_deployment_authentications.rb

```diff
@@ -0,0 +1,40 @@
+class CreateDeploymentAuthentications < ActiveRecord::Migration[5.0]
+  def change
+    create_table :deployment_authentications do |t|
+      t.string :name
+      t.boolean :challenge
+      t.boolean :login
+      t.string :kind
+      t.text :htpassd_users
+      t.text :ldap_id, :array => true, :default => []
+      t.text :ldap_email, :array => true, :default => []
+      t.text :ldap_name, :array => true, :default => []
+      t.text :ldap_preferred_user_name, :array => true, :default => []
+      t.string :ldap_bind_dn
+      t.string :ldap_bind_password
+      t.text :ldap_ca
```

Comment:
I'm not sure if it's something that needs to be stored in the DB, but we will be adding a variable for openshift_master_ldap_ca_file to transfer the CA file over to the host.